themes/docsy/layouts/partials/head-scripts.html

```diff
@@ -174,4 +174,34 @@
 
   return decodeURI(dc.substring(begin + prefix.length, end));
 }
+
+  function updateFilters() {
+      const activeFilters = Array.from(document.querySelectorAll('li.filter.active')).map(el => el.getAttribute("filter"))
+      window.history.pushState('', '', '?filters=' + activeFilters.join(","));
```

Comment:
Does this break if there's already query parameters in the URL?

Maybe we need to use `URLSearchParams` and add a `filters` param:

```
const p = new URLSearchParams(window.location.search)
p.set(filters, activeFilters.join(',')
```

Reply:
Fixed (but not like that for "reasons") :p